content/d3/concepts/styling/terms/style/style.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: '.style()'
+Description: 'Used to get or set inline properties of selected elememts.'
+Subjects:
+  - 'Data Visualization'
+  - 'Computer Science'
+Tags:
+  - 'D3'
+  - 'Style'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/computer-science'
+---
+
+the **`.style()`** method is used to set or get the inline style properties of selected elements. This method is useful for applying CSS styles to elements.
+
+## Syntax
+
+```pseudo
+selection.style("property", "value");
+```
+
+`.style()` funtion takes two arguements, first is the style name and second is the value to set to it.
```

Comment:
Why did you use `property` argument name instead of `style name`?

Reply:
@vrun1208 what you've decided? Can we work on the comments so we can move forward with the review.